### DISCUSSION TITLE
No TeX installation detected when trying to render pdf

### DISCUSSION THREAD

**Discussion by ericvmai** [quarto-dev/quarto-cli#2406] [Q&A]
I keep trying to render a quarto html and pdf but rstudio tells me: No TeX installation was detected.

But when I enter quarto install tool tinytex then it says it is already installed. When I try to produce a pdf of a different project - I can produce a PDF with no issue. Help!

https://github.com/hdadvisors/chesterfield-market-analysis/tree/quarto-transition

**Comment by cderv** [comment]
Thanks for the report. I have opened #2408 to investigate the issue and lock here. Please follow there. Thank you !
